src/google-classroom-extractor/google_classroom_extractor/mapping/assignment_submissions.py

```diff
@@ -44,8 +44,8 @@ def submissions_to_assignment_submissions_dfs(
         Status: The status of the submission in relation to the late acceptance policy
         SubmissionDateTime: The date and time of the assignment submission
         LMSUserIdentifier: A unique numeric identifier assigned to the user
-        CreateDate: Date this record was created
-        LastModifiedDate: Date this record was last updated
+        SourceCreateDate: Date this record was created in the LMS
+        SourceLastModifiedDate: Date this record was last updated in the LMS
```

Comment:
Don't we need all four columns in the output file?

Reply:
Yes, that was an early step to rename to "SourceXYZDate".  Still need to document and include the other two.